torchvision/prototype/transforms/_container.py

```diff
@@ -33,6 +33,7 @@ def extra_repr(self) -> str:
 
 
 class RandomChoice(Transform):
+    # datumbox: Shouldn't this be `transforms: Sequence[Callable]` for BC, similar to what we did for `Compose`?
```

Comment:
cc @vfdev-5

Reply:
I'll fix that

Reply:
https://github.com/pytorch/vision/pull/6488